### PR TITLE
removed note section

### PR DESCRIPTION
**Pull request by MaratMussabekov**
I believe this section better be removed as it does not contain any specific information and only confuses
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/6544

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @nam31, for your information

Thank you

File: windows/security/identity-protection/hello-for-business/hello-hybrid-key-whfb-provision.md [hunk 1]
```diff
@@ -57,9 +57,6 @@ The remainder of the provisioning includes Windows Hello for Business requesting
 > **This synchronization latency delays the user's ability to authenticate and use on-premises resources until the user's public key has synchronized to Active Directory.** Once synchronized, the user can authenticate and use on-premises resources.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-key-whfb-provision.md:57]:
```suggestion
> **This synchronization latency delays the user's ability to authenticate and uses on-premises resources until the user's public key has synchronized to Active Directory.** Once synchronized, the user can authenticate and use on-premises resources.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Not related but please also merge the additional edit.

**Comment by nam31** [comment]
@mapalko PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

File: windows/security/identity-protection/hello-for-business/hello-hybrid-key-whfb-provision.md [hunk 2]
```diff
@@ -54,12 +54,9 @@ The remainder of the provisioning includes Windows Hello for Business requesting
 
 > [!IMPORTANT]
 > The minimum time needed to synchronize the user's public key from Azure Active Directory to the on-premises Active Directory is 30 minutes. The Azure AD Connect scheduler controls the synchronization interval. 
-> **This synchronization latency delays the user's ability to authenticate and use on-premises resources until the user's public key has synchronized to Active Directory.** Once synchronized, the user can authenticate and use on-premises resources.
+> **This synchronization latency delays the user's ability to authenticate and uses on-premises resources until the user's public key has synchronized to Active Directory.** Once synchronized, the user can authenticate and use on-premises resources.
```
Comment (mapalko) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-key-whfb-provision.md:57]: I think that this should still be "use" and not "uses"

**Review by mapalko — CHANGES_REQUESTED** [review]

File: windows/security/identity-protection/hello-for-business/hello-hybrid-key-whfb-provision.md (hunk 2 above)
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-key-whfb-provision.md:57]:
```suggestion
> **This synchronization latency delays the user's ability to authenticate and use on-premises resources until the user's public key has synchronized to Active Directory.** Once synchronized, the user can authenticate and use on-premises resources.
```

Comment (JohanFreelancer9) [reply windows/security/identity-protection/hello-for-business/hello-hybrid-key-whfb-provision.md:57]: @mapalko - Thank you. After reading the sentence again, I realize that you are right. Apologies, @MaratMussabekov.

**Review by illfated — APPROVED** [review]
Looks good to me.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**